blivet/osinstall.py

```diff
@@ -1094,6 +1099,653 @@ def get(self, key, default=None):
         return self.mappings.get(key, default)
 
 
+class InstallerStorageConfig(Blivet):
```

Comment:
I'm not married to the name I originally gave it, so I can change it, sure. I'll just call it InstallerStorage.